CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ IMPROVEMENTS:
    rolling update could remove an unnecessary amount of allocations [GH-3070]
  * api: Redact Vault.Token from AgentSelf response [GH-2988]
  * cli: node-status displays node version [GH-3002]
+ * cli: Add status command with autocomplete [GH-3047]
```

Comment:
Add status command which takes any identifier and routes to the appropriate status command.